### PR TITLE
Implement swipe on Tabs header

### PR DESCRIPTION
**Pull request by igor-ribeiro**
#267

### CODE REVIEW

**Comment by marcosmoura** [comment]
I think this option should be default and applied anyway. What do you think?

**Comment by marcosmoura** [comment]
@igor-ribeiro Can you change that?

**Comment by igor-ribeiro** [comment]
@marcosmoura I will!

**Comment by igor-ribeiro** [comment]
I did it. The only problem is when you resize it on desktop the scrollbar becomes visible. This happens because my hack to hide the scrollbar uses negative margin, but on mobile there is no scrollbar like on desktop, so there is no need for the negative margin.

File: src/components/mdTabs/mdTabs.vue [hunk 1]
```diff
@@ -40,6 +42,7 @@
       mdFixed: Boolean,
       mdCentered: Boolean,
       mdRight: Boolean,
+      mdScrollableNavigation: Boolean,
```
Comment (marcosmoura) [inline src/components/mdTabs/mdTabs.vue]: Can you make this property true by default?

**Comment by dohomi** [comment]
It would be great if the tab header would have a left/right navigation arrows visible here: https://material.angularjs.org/latest/demo/tabs. Thanks for this improvement!

File: src/components/mdTabs/mdTabs.vue (hunk 1 above)
Comment (igor-ribeiro) [inline src/components/mdTabs/mdTabs.vue]: I can actually removed since it is the default behavior now.

**Comment by igor-ribeiro** [comment]
It would be very nice. What do you think, @marcosmoura ?

**Comment by Morgul** [comment]
@igor-ribeiro with regards to the arrows, I think they should clip the tabs under them; the transparency is a little hard to read. But _personally_, I really like that idea. I've wanted that functionality a few times in a tab component.

**Comment by igor-ribeiro** [comment]
@Morgul I agree!

**Comment by igor-ribeiro** [comment]
tabs-navigation

**Comment by marcosmoura** [comment]
I think this is awesome, but can you create a different PR for that?

**Comment by marcosmoura** [comment]
@igor-ribeiro Can you sync your branch with the latest develop?

**Comment by marcosmoura** [comment]
@igor-ribeiro ping

**Comment by igor-ribeiro** [comment]
@marcosmoura Done!

**Comment by marcosmoura** [comment]
Sounds good!
I'm just holding this until v0.8.0 release.
Thank you so much for this PR.

**Comment by sinisarudan** [comment]
Hi, is this still not solved?
Namely, even after adding the prop:
<md-tabs md-swipeable>, md-tabs looks like this.
tabs being cut-out, squeezed and not swipeable.
image

**Pull request merged**